sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -65,7 +65,7 @@ For example, `https://<my-custom-subdomain>.cognitiveservices.azure.com/`.
 Install the Azure Text Analytics client library for Python with [pip][pip]:
 
 ```bash
-pip install azure-ai-textanalytics
+pip install azure-ai-textanalytics --pre
```

Comment:
we need to update the Note below. Maybe be specific on what version you're getting with/without the --pre flag?

Reply:
v good point thankns

Reply:
just added a note about this. in the note i explicitly mention 5.0.0 as the latest GA version of the library, since I'm mentioning that the GA version only includes support of v3.0, and this behavior is tied to that version